Website/src/rendering/src/components/Checkout/CartSummary.tsx

```diff
@@ -4,7 +4,7 @@ const CartSummary = (): JSX.Element => {
   const { order } = useOcCurrentOrder();
   const subtotal = order && (
     <p>
-      Subtotal ({order.LineItemCount} items): ${order.Subtotal}
+      Subtotal ({order.LineItemCount} items): <strong>${order.Subtotal}</strong>
```

Comment:
@NellyKostadinova 
From https://developer.mozilla.org/en-US/docs/web/html/element/strong
> The `<strong>` element is for content that is of "strong importance," including things of great seriousness or urgency (such as warnings).
> Typically this element is rendered by default using a bold font weight. However, it should not be used to apply bold styling; use the CSS `font-weight` property for that purpose.

I personally prefer to use HTML elements for their semantic purpose rather than the default styling they provide. It also helps for accessibility and screen readers.